go/libkb/passphrase_stream.go

```diff
@@ -87,6 +87,23 @@ func (ps *PassphraseStream) SetGeneration(gen PassphraseGeneration) {
 	ps.gen = gen
 }
 
+func newPassphraseStreamPE(pwhash, eddsa []byte) (*PassphraseStream, error) {
```

Comment:
confusing abbreviation, maybe it's better for a longer name.

Can you make a type here rather than `[]byte`. I'm remember bug 3964 which was so much effort to recover from. That was a result of using `[]byte`s in the login path rather than the specialized `LKS...` types.

Reply:
do you mind if I change typing in another PR? I'd like to introduce more types to `passphrase_stream.go` but this would require additional changes to consumers of that API.

or is there some specific small change I should do here right now?

Reply:
don't follow. can't you change this signature and then switch to bytes one layer down?

Reply:
so we would have something like:
```
type PWHash []byte
```
and in `RetrievePwhashEddsaPassphraseStream`:
```
pwHashSecret, err := ss.RetrieveSecret(mctx, formatPPSSecretStoreIdentifier(username, ssPwhashSuffix))
pps, err := newPassphraseStreamFromPwhAndEddsa(PWHash(pwHashSecret.Bytes()), EdDSA(edDSASecret.Bytes())
```
I thought that does not buy us much

Reply:
what you have, with two `[]byte` params, is just really dangerous. Get that slightly wrong, either by transposition, or the wrong type of data, and you'll have a very difficult to diagnose, debug and fix bug on your hand. See Bug 3964

Reply:
thanks!